goal_publisher:
  ros__parameters:
    goal_frame: /map
    goal_x: 0.
    goal_y: 1.
    goal_yaw_in_degree: 45. 

footprint_publisher:
  ros__parameters:
    circle_footprint:
      #If this is true, the robot will assume 
      #to have circular footprint
      use_circle_footprint: false 
      radius: 0.2 #in m
    polygon_footprint:
      frame_id : /base_footprint
      footprint_x: [-0.2, 0.2, 0.2,-0.2] #in m
      footprint_y: [0.1, 0.1,-0.1,-0.1] #in m

dwa_planner_node:
  ros__parameters:
    MAX_LINEAR_VEL: 1.
    min_linear_vel: 0. 
    max_yaw_rate: 90. #in degree
    max_acceleration: 1.
    max_deceleration: 1.
    max_yaw_acceleration: 45